recipe/meta.yaml

```
@@ -1,27 +1,27 @@
 {% set name = "aws-c-cal" %}
-{% set version = "0.6.9" %}
+{% set version = "0.5.20" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/awslabs/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 46defd807034ca9d873cc1459bc60f0bb4662029f7120db1af464421907fb480
+  sha256: acc352359bd06f8597415c366cf4ec4f00d0b0da92d637039a73323dd55b6cd0
 
 build:
-  number: 3
+  number: 0
   run_exports:
     - {{ pin_subpackage("aws-c-cal", max_pin="x.x.x") }}
 
 requirements:
   build:
     - cmake !=3.19.0,!=3.19.1
     - {{ compiler('c') }}
-    - ninja
+    - ninja-base
   host:
-    - aws-c-common
-    - openssl  # [not win and not osx]
+    - aws-c-common 0.8.5
+    - openssl {{ openssl }}
```

```suggestion
    - openssl {{ openssl }}  # [not win and not osx]
```
OpenSSL is only needed on systems other than Windows and macOS. On macOS, it uses the Security Framework (https://github.com/awslabs/aws-c-cal/blob/main/source/darwin/securityframework_rsa.c#L10) and on Windows it uses https://learn.microsoft.com/en-us/windows/win32/api/bcrypt/ (see https://github.com/awslabs/aws-c-cal/blob/main/source/windows/bcrypt_rsa.c#L15-L15).

And conda-build complains that openssl isn't used:
```
   WARNING (aws-c-cal): run-exports library package defaults/osx-64::openssl==3.0.12=hca72f7f_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```